src/ui/theme.tsx

```diff
@@ -201,6 +192,30 @@ export function createGetViewPortConfig(params: {
     return { getViewPortConfig };
 }
 
+function getPalette(themeId: typeof THEME_ID, paletteOverrides?: Object): PaletteBase {
+    var palette: PaletteBase;
+    switch (themeId) {
+        case "onyxia":
+            palette = defaultPalette;
+            break;
+        case "france":
+            palette = francePalette;
+            break;
+        case "ultraviolet":
+            palette = ultravioletPalette;
+            break;
+        case "verdant":
+            palette = verdantPalette;
+            break;
+    }
+
+    if (paletteOverrides) {
+        _.merge(palette, paletteOverrides);
```

Comment:
This is destructive for the default palette.  
In practice it's no big deal but it's an undesired side effect.